tests/http/test_http_parser.py

```diff
@@ -16,13 +16,23 @@
 
 from proxy.http import httpStatusCodes, httpMethods
 from proxy.http.parser import HttpParser, httpParserTypes, httpParserStates
+from proxy.http.exception import HttpProtocolException
 
 
 class TestHttpParser(unittest.TestCase):
 
     def setUp(self) -> None:
         self.parser = HttpParser(httpParserTypes.REQUEST_PARSER)
 
+    def test_issue_127(self) -> None:
+        with self.assertRaises(HttpProtocolException):
+            self.parser.parse(CRLF)
+
+        with self.assertRaises(HttpProtocolException):
```

Comment:
You forgot to change exception type in test.

Reply:
Sorry, It's my fault. Fixed already.